static/css/basic.css

```diff
@@ -708,3 +708,16 @@ table.dataTable input[type=text] {
     height: 25px;
     border: 2px solid var(--navbar-color);
 }
+
+/* Operations */
+
+#opBtn {
```

Comment:
This and `#saveAgentButton` could just be a common class selector that is used instead of styling each ID selector separately

Reply:
Addressed in more recent commits